k8s-artifacts/mg-adapter/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Deployment for Adapter
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mg-adapter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mg-adapter
+  template:
+    metadata:
+      labels:
+        app: mg-adapter
+    spec:
+      containers:
+        - name: mg-adapter
+          volumeMounts:
+            - mountPath: /home/wso2/security/localhost.key
+              subPath: localhost.key
+              name: adapter-key-vol
+            - mountPath: /home/wso2/security/localhost.pem
+              subPath: localhost.pem
+              name: adapter-pem-vol
+            - mountPath: /home/wso2/conf/config.toml
+              subPath: config.toml
+              name: config-toml-vol
+            - mountPath: /home/wso2/conf/log4j2.properties
+              subPath: log4j2.properties
+              name: log4j2-vol
+            - mountPath: /home/wso2/conf/log_config.toml
+              subPath: log_config.toml
+              name: logconfig-toml-vol
+          image: wso2/mg-adapter:4.0.0-m1
```

Comment:
Let add resource limits as well

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf